app/app.js

```diff
@@ -186,7 +186,7 @@ angular.module('app', ['angular-md5'])
       // Get the collection containing this comic
       vm.expandedCollection = _.find(collections, function(collection, index) {
         currentComicIndexInCollection = collection.comicIds.indexOf(currentComic.id);
-        if (currentComicIndexInCollection > -1) {
+        if (currentComicIndexInCollection === Math.abs(currentComicIndexInCollection)) { //there's probably a better way to do this
```

Comment:
I don't understand what this is doing now

Reply:
Have refactored to _.includes()
My thinking for checking a negative number is to return the absolute value.  Any positive number, or 0 will always exactly equal it's absolute value, but the absolute value of -1 is 1, so that was my way of checking that it wasn't a negative number.